crates/telio-model/src/features.rs

```diff
@@ -736,6 +740,25 @@ mod tests {
             let expected = FeatureWireguard::default();
             assert_eq!(actual, expected);
         }
+
+        #[test]
+        fn test_hide_ips() {
```

Comment:
I recently started questioning the relevance of tests that are this granular since it's mostly testing that `serde` works as expected. The way the tests in this module is structured now is one test to set all values to non-default values in json and then tests to make sure the correct defaults are set when objects are empty. So this would just need to be added to the big json block and that should be it. At least that's my opinion

Reply:
I find it to be way more easy to understand this test which I can actually fit in my head. The huge blob of json (and the accompanying huge struct expression) are enough for me to lose context.

Reply:
I can understand the big json blob being a bit too big too fit in ones head. I'm not against testing the way you're doing but in that case I would like to see it done for all fields (obviously not in this PR though), maybe with some parameterization and json construction so we don't have to manually write these tests for all feature options. I started adding tests like that manually before settling on the current structure but it was becoming too much boilerplate after only doing a fraction of the existing fields

Reply:
I will create a ticket to look into this in future.